_includes/manuals/3.6/1.2_release_notes.md

```diff
@@ -43,6 +43,15 @@ Starting 2023-03-01 Puppet 6 is end of life. Users should upgrade to Puppet 7.
 
 ### Release Notes
 
+### Release notes for 3.6.1
```

Comment:
I noticed this is the same header as above, but I think that's been this way for a long time so I didn't address it.